server/src/main/java/org/apache/iotdb/db/query/timegenerator/ServerTimeGenerator.java

```diff
@@ -33,8 +36,6 @@
 import org.apache.iotdb.tsfile.read.query.timegenerator.TimeGenerator;
 import org.apache.iotdb.tsfile.read.reader.IBatchReader;
 
-import java.io.IOException;
-
 /**
  * A timestamp generator for query with filter. e.g. For query clause "select s1, s2 form root where
  * s3 < 0 and time > 100", this class can iterate back to every timestamp of the query.
```

Comment:
For query clause "select s1, s2 **form** root -> For query clause "select s1, s2 **from** root

Reply:
Thanks!  Good catch : )